### ISSUE TITLE
Hello world example not working

### ISSUE THREAD

**Issue by coder966** [mandarineorg/mandarinets#251]
**[Details]**
I just copied the hello world example located at: https://www.mandarinets.org/docs/master/mandarine/hello-world
and when I run it it says:

```
PS C:\Users\name\Desktop\deno-test> deno run -c tsconfig.json --allow-net --allow-read --allow-env .\app.ts 
Check file:///C:/Users/name/Desktop/deno-test/app.ts
error: TS2345 [ERROR]: Argument of type 'T | undefined' is not assignable to parameter of type 'T | PromiseLike<T>'.
  Type 'undefined' is not assignable to type 'T | PromiseLike<T>'.
      resolve(value);
              ~~~~~
    at https://raw.githubusercontent.com/mandarineorg/mandarine-postgres/v1.1.0/utils.ts:94:15
```

**[Environment]**
 - Mandarine version: 2.3.0
 - Deno version: 1.6.3 (release, x86_64-pc-windows-msvc)
 - Deno v8 version: 8.8.294
 - Deno typescript version: 4.1.3

**Comment by coder966** [comment]
I was able to make it run by adding `"strict": false,` to `tsconfig.json` even though it is not mentioned in the official docs as a necessary config.

see: https://www.mandarinets.org/docs/master/mandarine/main-configuration

**Comment by andreespirela** [comment]
Hello there. This issue is solved in Master and will be resolved in the next release which is 2.4.0
